src/box/index.h

```diff
@@ -252,6 +252,15 @@ box_tuple_extract_key(box_tuple_t *tuple, uint32_t space_id,
 
 /** \endcond public */
 
+/**
+ * Get packed position of tuple in index to pass it to box_select
+ * (multikey and func indexes are not supported).
+ */
+int
+box_index_tuple_position(uint32_t space_id, uint32_t index_id,
+			 const char *tuple, const char *tuple_end,
+			 const char **pos, const char **pos_end);
```

Comment:
We should use `packed_pos` here as well as in `lbox_select`, `lbox_normalize_position`, I think.
`pos` for packed position is okay to use only in Lua where there's no such thing as unpacked position.

Reply:
Done